src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_vars.py

```diff
@@ -608,13 +613,15 @@ def change_attr_expression(frame, attr, expression, dbg, value=SENTINEL_VALUE):
             if result is not dbg.plugin.EMPTY_SENTINEL:
                 return result
 
-        if attr[:7] == "Globals":
-            attr = attr[8:]
+        if attr[:7] == "Globals" or scope == "globals":
+            attr = attr[8:] if attr.startswith("Globals") else attr
```

Comment:
Given that the `attr[8:]` range starts at 8, is there one more required character in the `"Globals"` prefix that should be added to the `startswith` check?

Reply:
I don't think so as the if line above doesn't check for that. I didn't add this part so I'm not sure what the 8th character is supposed to be